k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapListTest.java

```diff
@@ -11,33 +14,129 @@
 import static org.junit.Assert.fail;
 
 public class ImapListTest {
-    @Test public void testImapListMethods() throws IOException {
+
+    private ImapList buildSampleList() {
```

Comment:
Helper methods are usually not very interesting. That's why I like to keep them at the end of the test class (out of sight).